code/game/objects/structures/ai_core.dm

```diff
@@ -25,218 +31,191 @@
+					ticker.mode.remove_antag_for_borging(M.brainmob.mind)
+					remove_servant_of_ratvar(M, TRUE)
+					M.forceMove(src)
+					brain = M
+					user << "<span class='notice'>Added a brain.</span>"
 					icon_state = "3b"
-				else
+					return
+
+				if(istype(P, /obj/item/weapon/crowbar) && brain)
+					playsound(loc, 'sound/items/Crowbar.ogg', 50, 1)
+					user << "<span class='notice'>You remove the brain.</span>"
+					brain.forceMove(loc)
+					brain = null
 					icon_state = "3"
-				new /obj/item/stack/sheet/rglass(loc, 2)
-				return
+					return
 
-			if(istype(P, /obj/item/weapon/screwdriver))
-				playsound(loc, 'sound/items/Screwdriver.ogg', 50, 1)
-				user << "<span class='notice'>You connect the monitor.</span>"
-				new /mob/living/silicon/ai (loc, laws, brain)
-				feedback_inc("cyborg_ais_created",1)
-				qdel(src)
-				return
+			if(GLASS_CORE)
+				if(istype(P, /obj/item/weapon/crowbar))
+					playsound(loc, 'sound/items/Crowbar.ogg', 50, 1)
+					user << "<span class='notice'>You remove the glass panel.</span>"
+					state = CABLED_CORE
+					if(brain)
+						icon_state = "3b"
+					else
+						icon_state = "3"
+					new /obj/item/stack/sheet/rglass(loc, 2)
+					return
+
+				if(istype(P, /obj/item/weapon/screwdriver))
+					playsound(loc, 'sound/items/Screwdriver.ogg', 50, 1)
+					user << "<span class='notice'>You connect the monitor.</span>"
+					new /mob/living/silicon/ai (loc, laws, brain)
+					feedback_inc("cyborg_ais_created",1)
+					qdel(src)
+					return
 	return ..()
 
 
 /obj/structure/AIcore/deconstruct(disassembled = TRUE)
-	if(state == 4)
+	if(state == GLASS_CORE)
 		new /obj/item/stack/sheet/rglass(loc, 2)
-	if(state >= 3)
+	if(state >= CABLED_CORE)
 		new /obj/item/stack/cable_coil(loc, 5)
 	if(circuit)
 		circuit.forceMove(loc)
 		circuit = null
-	new /obj/item/stack/sheet/plasteel( loc, 4)
+	new /obj/item/stack/sheet/plasteel(loc, 4)
 	qdel(src)
 
 /obj/structure/AIcore/deactivated
 	name = "inactive AI"
 	icon = 'icons/mob/AI.dmi'
 	icon_state = "ai-empty"
 	anchored = 1
-	state = 20//So it doesn't interact based on the above. Not really necessary.
+	state = GLASS_CORE
 
 /obj/structure/AIcore/deactivated/attackby(obj/item/A, mob/user, params)
-	if(istype(A, /obj/item/device/aicard))//Is it?
+	if(istype(A, /obj/item/device/aicard) && state == GLASS_CORE)
 		A.transfer_ai("INACTIVE","AICARD",src,user)
-	else if(istype(A, /obj/item/weapon/wrench))
-		playsound(loc, 'sound/items/Ratchet.ogg', 50, 1)
-		user.visible_message("[user] [anchored ? "fastens" : "unfastens"] [src].", \
-					 "<span class='notice'>You start to [anchored ? "fasten [src] to" : "unfasten [src] from"] the floor...</span>")
-		switch(anchored)
-			if(0)
-				if(do_after(user, 20, target = src))
-					user << "<span class='notice'>You fasten the core into place.</span>"
-					anchored = 1
-			if(1)
-				if(do_after(user, 20, target = src))
-					user << "<span class='notice'>You unfasten the core.</span>"
-					anchored = 0

```

Comment:
Hey, I know it's not your code, but `atom/proc/transfer_ai(` below is missing a `/` at the beginning.